dev/tools/lib/roll_dev.dart

```diff
@@ -16,13 +16,12 @@ const String kIncrement = 'increment';
 const String kX = 'x';
 const String kY = 'y';
 const String kZ = 'z';
+const String kCommit = 'commit';
 const String kHelp = 'help';
 
-void main(List<String> args) {
-  // Set the cwd to the repo root, since we know where this script is located.
-  final Directory scriptLocation = new Directory(Platform.script.toFilePath());
-  Directory.current = scriptLocation.parent.parent.parent;
+const String kUpstreamRemote = 'git@github.com:flutter/flutter.git';
```

Comment:
What if they're using https access instead of .git access?

Reply:
Our contributing docs say to use .git, I'm comfortable saying that if you want to publish you have to follow that script to the letter. :-)

Reply:
Works for me. :-)